src/site/markdown/changelog.md

```diff
@@ -6,6 +6,15 @@ The changelog should give you a quick overview of what change between the differ
 
 ### New Features
 
+- [5](https://github.com/FHOOEAIST/imaging/issues/5) - Added the mesh module with multiple storage implementations for JavaModel3D class (OBJ, OFF, PLY, STL) as well as a Constrained Delaunay implementation 
+
+### Dependencies
+- [5](https://github.com/FHOOEAIST/imaging/issues/5) - Added a dependency to [jDelaunay](https://github.com/orbisgis/jdelaunay) for constrained delaunay triangulation
+
+## Version 1.1.0
```

Comment:
I will add this here, as git does not show the heading above ... 
It should be Version 1.2.0